src/hgvs/utils/altseqbuilder.py

```diff
@@ -326,7 +340,7 @@ def _setup_incorporate(self):
                     result = cds_start - 1
                 else:  # cds/intron
                     if pos.offset <= 0:
-                        result = (cds_start - 1) + pos.base - 1
+                        result = (cds_start - 1) + pos.base + pos.offset - 1
```

Comment:
Do you know if this line is covered by tests prior to this PR? It's suspicious to me that a bug fix of this magnitude doesn't have broader impacts.

Reply:
I was a little surprised too but if I remove it the following tests fail:
```
    def test_map_of_dup_intron_exon_boundary(self):
        hgvs_c = "NM_024529.4:c.132-2_132-1dup"
        hgvs_p = "NP_078805.3:p.(Thr45GlyfsTer65)"
    
        var_c = self.hp.parse_hgvs_variant(hgvs_c)
        var_p = self.am.c_to_p(var_c)
    
>       self.assertEqual(str(var_p), hgvs_p)
E       AssertionError: 'NP_078805.3:p.(Thr45ArgfsTer65)' != 'NP_078805.3:p.(Thr45GlyfsTer65)'
E       - NP_078805.3:p.(Thr45ArgfsTer65)
E       ?                     ^^^
E       + NP_078805.3:p.(Thr45GlyfsTer65)
E       ?                     ^^^

tests/test_hgvs_assemblymapper.py:210: AssertionError

    def test_map_of_dup_intron_exon_boundary_rc(self):
        hgvs_c = "NM_004380.2:c.3251-1dup"
        hgvs_p = "NP_004371.2:p.(Ile1084SerfsTer3)"
    
        var_c = self.hp.parse_hgvs_variant(hgvs_c)
        var_p = self.am.c_to_p(var_c)
    
>       self.assertEqual(str(var_p), hgvs_p)
E       AssertionError: 'NP_004371.2:p.(Ile1084MetfsTer3)' != 'NP_004371.2:p.(Ile1084SerfsTer3)'
E       - NP_004371.2:p.(Ile1084MetfsTer3)
E       ?                       ^ ^
E       + NP_004371.2:p.(Ile1084SerfsTer3)
E       ?                       ^ ^

tests/test_hgvs_assemblymapper.py:227: AssertionError

    def test_map_of_ins_intron_exon_boundary(self):
        hgvs_c = "NM_004380.2:c.3251-1_3251insA"
        var_c = self.hp.parse_hgvs_variant(hgvs_c)
        var_p = self.vm.c_to_p(var_c)
>       self.assertEqual(str(var_p), "NP_004371.2:p.(Ile1084AsnfsTer3)")
E       AssertionError: 'NP_004371.2:p.(Phe1085LeufsTer2)' != 'NP_004371.2:p.(Ile1084AsnfsTer3)'
E       - NP_004371.2:p.(Phe1085LeufsTer2)
E       ?                ^^    ^^^^     ^
E       + NP_004371.2:p.(Ile1084AsnfsTer3)
E       ?                ^^    ^^^^     ^

tests/test_hgvs_variantmapper.py:125: AssertionError
```

Reply:
@reece While working on my latest commit I found a way to pull the offset out of this calculation.

Reply:
With `+ pos.offset` in line 329 in main, tests still pass. The only way that can be true, I think, is that pos.offset is always 0 for all tests. This might be a gap in tests. (Nothing for you to do here.)